homeassistant/components/agent_dvr/camera.py

```diff
@@ -80,6 +80,7 @@ def __init__(self, device):
         self._removed = False
         self._attr_name = f"{device.client.name} {device.name}"
         self._attr_unique_id = f"{device._client.unique}_{device.typeID}_{device.id}"
+        self._attr_should_poll = True
```

Comment:
The camera base entity overwrites `should_poll` so it's correct to implement that property but probably in a separate PR.